integration/spadev0/util.go

```diff
@@ -164,7 +191,7 @@ func addErrorResults(
 		Task: task.Task{
 			Requester: requester,
 			Module:    "spadev0",
-			Metadata:  spadev0Metadata,
+			Metadata:  make(map[string]string),
```

Comment:
Wasnt sure which metadata to use for the error unless we want to pass it around

Reply:
hmm, have a look at the`addErrorResults` func in `integration/filplus/util.go` to see how it's done there. I originally copied from here but simplified as we only had one type - might be better to copy what they have.